src/py42/clients/users.py

```diff
@@ -27,7 +27,7 @@ def create_user(
         Args:
             org_uid (str): The org UID for the organization the new user belongs to.
             username (str): The username for the new user.
-            email (str, optional): The email for the new user. Defaults to None.
+            email (str): The email for the new user.
```

Comment:
remove `=None` from email in the args list, doing this won't be a breaking change if not including `email` already doesn't work.